### PR TITLE
Update python.html.markdown

### PR DESCRIPTION
**Pull request by codeNameMahi**
made some minor improvements to section 1:
- emphasized the different types
- added example that distinguishes between int and float
- added example of changing other types to string
- added warning to not confuse = with ==
- added example output for string formatting
- made formatting consistent ("# =>" instead of "#=>"

### CODE REVIEW

File: python.html.markdown [hunk 1]
```diff
@@ -107,26 +115,36 @@ not False  # => True
...
 "Hello " + "world!"  # => "Hello world!"
+
 # Strings can be added without using '+'
 "Hello " "world!"  # => "Hello world!"
 
 # ... or multiplied
 "Hello" * 3  # => "HelloHelloHello"
 
+# A number can be a string
+type("2") # => <type 'str'>
```
Comment (adambard) [inline python.html.markdown]: I think mentioning this make things more confusing. Remember that the reader is assumed to know at least one other programming language -- they probably understand that a string of number characters is considered a string.

Probably more useful to demonstrate that you can't just `+` a number onto a string for the benefits of php and javascript devs.

File: python.html.markdown [hunk 2]
```diff
@@ -35,22 +35,26 @@ allow you to write Python 3 code that will run on Python 2, so check out the Pyt
...
+# You have numbers, which are type int
 3  # => 3
+type(3) # => <type 'int'>
 
 # Math is what you would expect
-1 + 1  # => 2
-8 - 1  # => 7
-10 * 2  # => 20
-35 / 5  # => 7
+1 + 1  # addition => 2
```
Comment (geoffliu) [inline python.html.markdown]: I don't think it's necessary to explain what `+` does to most programmers. Nor the following three lines.

File: python.html.markdown [hunk 3]
```diff
@@ -67,23 +72,26 @@ allow you to write Python 3 code that will run on Python 2, so check out the Pyt
 # Enforce precedence with parentheses
 (1 + 3) * 2  # => 8
 
-# Boolean Operators
+# Boolean Operators, which are type bool
```
Comment (geoffliu) [inline python.html.markdown]: Boolean operators are not of type bool. The literals are, but the operators are not.

File: python.html.markdown [hunk 4]
```diff
@@ -67,23 +72,26 @@ allow you to write Python 3 code that will run on Python 2, so check out the Pyt
...
+0 == False # => True
+2 == True # => False
+1 == True # => True
 
 # negate with not
 not True  # => False
 not False  # => True
 
 # Equality is ==
+# careful not to confuse with =, which is used for assignment
```
Comment (geoffliu) [inline python.html.markdown]: A bit gratuitous for seasoned programmers.

**Comment by vendethiel** [comment]
@codeNameMahi any news?:)

**Comment by vendethiel** [comment]
Going to close this one for now :). Feel free to ping us here if you find some time working on it again! Thanks.

**Pull request closed without merging**